src/sas/sascalc/dataloader/data_info.py

```diff
@@ -714,7 +714,7 @@ def __init__(self, x=None, y=None, dx=None, dy=None, lam=None, dlam=None, isSesa
                 self.x_unit = '1/A'
                 self.y_unit = '1/cm'
         except: # the data is not recognized/supported, and the user is notified
-            raise(TypeError, 'data not recognized, check documentation for supported 1D data formats')
+            raise TypeError
 
```

Comment:
Apparently python 2 allowed the raising of a tuple, by 2to3 doesn't know about it:

http://stackoverflow.com/questions/9836756/why-raising-a-tuple-works-if-first-element-is-an-exception#9837750